docker/dev/.env.example

```diff
@@ -11,3 +11,4 @@ JWT_REFRESH_TOKEN_SECRET=
 RAFIKI_STREAM_SECRET=
 SENDGRID_API_KEY=
 FROM_EMAIL=
+SEND_EMAIL=
```

Comment:
Another environment variables is probably not needed. We can just check the `NODE_ENV`.

Reply:
a new env variable allows to enable send email on dev for testing purposes without affecting anything else

Reply:
Yes, we sometimes do need to send email in dev environment as well, to test somethings, so maybe an env variable is the best for now

Reply:
@raducristianpopa 
> I maintain my stance that an environment variable is unnecessary to control either or not an email should be sent. We already have a lot of environment variables. To test the templates, a straightforward script that sends an email would be a perfect fit.

I don't see any issues with having a lot of env variables - for sure we will have more.
Testing the template with a separate script will require additional time from the engineer to prepare it.
If an email is not sent in the testing process, how to ensure in development that send email functionality works properly and there are no issues?

Reply:
SO let's agree to disagree for now on this, and leave it in .env :)

Reply:
@raducristianpopa  I get your point as well, but I am voting for the env variable for this. I think it's not that much of an overhead in configuring environments of our application, and I think it might be better than having the process context itself as the deciding factor.

Reply:
Unresolved this for @raducristianpopa  to see